raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Environmentally sensitive areas 
+(51) [1] Where the Minister considers it necessary, he may by order published in the Gazette, declare any area to be environmentally sensitive under this Act. [2] Management of environmental sensitive areas under this section shall be done in accordance with procedures prescribed by the Minister. [3] Any person who fails, neglects or refuses to comply with procedures prescribed to regulate environmentally sensitive areas commits an offence.
+
+Identification of environmentally sensitive areas 
+(52) Without prejudice to section 51, the following land shall constitute environmentally sensitive areas for the purposes of this Act: [a] swamps; [b] any area declared as environmentally sensitive by any local government authority; [c] area designated by the Council as prone to soil erosion; [d] land designated by the Council as lands where landslides have occurred or are likely to occur; [e] all areas that have been closed by the Minister to livestock keeping, occupation, cultivation and other specified activities; [t] area on slopes with a gradient exceeding any angle which the Minister shall, after taking account of proper scientific advice, specify; [g] arid and semi-arid lands; [h] land specified by the Council as land which should not be developed on account of its fragile nature or of its environmental significance; and [i] land declared under any other written law to be environmentally sensitive area or hazardous land.
+
+Environmental conditions to village lands 
+(53) Apart from the terms and conditions prescribed by section 29 of the Village Land Act, 1999 regarding village lands or on land sharing arrangement between pastoralists and agriculturalists in terms of section 58 of the Village Land Act, 1999, the Minister may prescribe any other additional environmental protection conditions to be complied with by the grantees of customary rights of occupancy. 
+
+Declaration of protected river, riverbanks, lake or lakeshore and shoreline 
+(54) [1] Notwithstanding any other written law for the time being regulating rivers, riverbanks, lakes or lakeshores and shoreline, the Minister may, in consultation with other relevant Ministries, by notice published in the Gazette: [a] declare a river, riverbank, lake, or lakeshore and shoreline a protected area for the purposes of this Act; and [b] impose any restrictions as he considers necessary for the protection of the river, riverbank, lake or lakeshore and shorelines from environmental degradation. [2] While exercising powers under subsection [1], the Minister shall have regard to: [a] the geographical size of the river, riverbank, lake or lakeshore and shoreline; [b] the interests of the communities resident around the river, riverbank, lake or lakeshore and shoreline concerned; and any advice that may be given by sector ministries. 
```

Comment:
Please replace `; and any advice` with `; and [c] any advice`.

I will resume my review at the next article later.